lib/ansible/plugins/loader.py

```diff
@@ -874,8 +874,8 @@ def get_with_context(self, name, *args, **kwargs):
                 obj = instance
             except TypeError as e:
                 if "abstract" in e.args[0]:
-                    # Abstract Base Class.  The found plugin file does not
-                    # fully implement the defined interface.
+                    # Abstract Base Class or incomplete plugin, don't load
+                    display.v('Returning not found on "%s" as it has abstract methods and is not a working plugin' % name)
```

Comment:
I think the Exception rendered as a string has more info that would be helpful to plugin developers (i.e. the specific un-implemented methods)

Reply:
will add